src/Microsoft.Health.Dicom.Functions/Registration/ServiceCollectionExtensions.cs

```diff
@@ -101,5 +108,43 @@ public static IMvcBuilder AddJsonSerializerOptions(this IMvcBuilder builder, Act
             builder.Services.Configure(configure);
             return builder;
         }
+
+        private static IServiceCollection AddTriggerAuthentication(this IServiceCollection services, SecurityConfiguration config)
```

Comment:
Unfortunately, we don't yet today have a package for shared ASP.NET Core dependencies, so I copied a subset of the logic from the API project here.